ang/civicase-base/services/case-management-workflow.service.js

```diff
@@ -1,5 +1,5 @@
 (function (_, angular) {
-  var module = angular.module('workflow');
+  var module = angular.module('civicase-base');
```

Comment:
Added the details now
> 4. `case-management-workflow.service.js` has been moved under `civicase-base` module, because it is now used outside of Workflows module also.